docs/development/devtools.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Devtools
+description: ✨ How to use Spicetify devtools.
+---
+
+## How to enable them
+TBA
+
+## How to open them
+TBA
+
+## Useful console commands
+TBA
```

Comment:
I'd remove this file and simply create an issue on the repo so that we can create it later on. I do not feel comfortable pushing this to main.